### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by zoranmccs** [electerm/electerm#3051] [Ideas]
First of all, I really like electerm. Thank you very much for your hard work.

My ideas are focused on UI/UX:

- Bookmarks: margins between connection label and edit icon can be bigger
- Bookmarks: Indentation when bookmark is in a folder can be smaller
- Bookmarks: Only show bookmark label, details can be in a tooltip (baloon) on mouseover, it will be much more readable
- Tabs: Only show connection name (label), with bigger margin between icons and connection name (they are overlaping at the moment)
- SSH: when you close SSH connection by typing exit or CTRL+D in the terminal, dialog Terminal connection lost appears, and it's not in the focus, so we must use mouse to close it and not with the Return key, which will be faster
- Options: Add the option to automatically close terminal window on exit (without Terminal connection lost dialog)

Once again, thank you and greetings from Serbia.

**Comment by zxdong262** [comment]
Thank you for the feedback, would try to improve in next release
